recipes/mumemto/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "mumemto" %}
+{% set version = "1.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/vikshiv/mumemto/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: e0a42fd5d3afde6366dbc43bb16cc38b5a0c87468a4c6978c4225d361d789f95
+
+build:
+  number: 0
+  pkg_format: '.conda'
+  run_exports:
+    - {{ pin_subpackage(name, max_pin='x.x') }}
+  script:
+    - mkdir build
+    - cd build
+    - cmake ..
+    - make install
+    - cd ..
+    - mkdir -p $PREFIX/bin
+    - mkdir -p $SP_DIR/mumemto
+    - cp build/mumemto $PREFIX/bin/mumemto_exec
+    - cp build/bin/newscanNT.x $PREFIX/bin/
+    - cp mumemto/*.py $SP_DIR/mumemto/
+    - cp mumemto/mumemto $PREFIX/bin/
```

Comment:
Probably you should/could use CMAKE_PREFIX_PATH ?! You use `make install` anyway.